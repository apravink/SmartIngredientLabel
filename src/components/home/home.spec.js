import React from "react";
import Home from "./home";

import renderer from "react-test-renderer";

describe("rendering", () => {
  test("renders correctly", () => {
    const result = renderer.create(<Home />).toJSON();
    console.log(result);
    expect(result).toMatchSnapshot();
  });
});
