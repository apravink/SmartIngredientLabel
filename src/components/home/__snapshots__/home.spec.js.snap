// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#42f4df",
        "flex": 2,
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#ddd",
        "flex": 8,
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#42f4df",
        "borderColor": "white",
        "borderStyle": "solid",
        "borderTopWidth": 3,
        "flex": 2,
        "width": "100%",
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#464F51",
          "borderColor": "white",
          "borderRadius": 50,
          "borderStyle": "solid",
          "borderWidth": 3,
          "height": 80,
          "justifyContent": "center",
          "overflow": "visible",
          "top": 3,
          "width": 80,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        onPress={undefined}
        style={
          Array [
            false,
            false,
            false,
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "justifyContent": "center",
            },
            undefined,
          ]
        }
        underlayColor="white"
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#FFC43D",
                "fontSize": 65,
              },
              Array [
                Object {
                  "backgroundColor": "transparent",
                },
                undefined,
              ],
              Object {
                "fontFamily": "Material Design Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          ï„€
        </Text>
      </View>
    </View>
  </View>
</View>
`;
